extensions/interactions/AlgebraicExpressionInput/AlgebraicExpressionInput.py

```diff
@@ -20,20 +20,26 @@
 
 from extensions.interactions import base
 
+from typing import List
+
+MYPY = False
+if MYPY:  # pragma: no cover
+    from extensions import domain
+
 
 class AlgebraicExpressionInput(base.BaseInteraction):
     """Interaction for algebraic expression input."""
 
-    name = 'Algebraic Expression Input'
-    description = 'Allows learners to enter algebraic expressions.'
-    display_mode = base.DISPLAY_MODE_INLINE
-    is_trainable = False
-    _dependency_ids = ['guppy', 'nerdamer']
-    answer_type = 'AlgebraicExpression'
-    can_have_solution = True
-    show_generic_submit_button = True
+    name: str = 'Algebraic Expression Input'
```

Comment:
Are these attributes `Final`?
Are we changing the value of these anywhere?

ditto for other interactions

Reply:
These attributes ae already defined in base.py